README.md

```diff
@@ -277,6 +277,7 @@ General users should install this tool like any other node module.
 * [`symbol-bootstrap run`](docs/run.md) - It boots the network via docker using the generated `docker-compose.yml` file and configuration. The config and compose methods/commands need to be called before this method. This is just a wrapper for the `docker-compose up` bash call.
 * [`symbol-bootstrap start`](docs/start.md) - Single command that aggregates config, compose and run in one line!
 * [`symbol-bootstrap stop`](docs/stop.md) - It stops the docker-compose network if running (symbol-bootstrap started with --detached). This is just a wrapper for the `docker-compose down` bash call.
+* [`symbol-bootstrap upgradeVotingKeys`](docs/upgradeVotingKeys.md) - It upgrades the voting keys and files when required.
```

Comment:
Unsure, the command creates the file when necessary, it doesn't register in the network. For that, the link command is required (or symbol cli)

Reply:
Unsure, the command creates the file when necessary, it doesn't register in the network. For that, the link command is required (or symbol cli)

Reply:
hehe...maybe ``register`` is not a good name either.
``upgradeVotingKeys`` doesnt make much sense to me since voting key/files are not upgradable. 
 @segfaultxavi thoughts?
 maybe ``addVotingKeys`` or ``createVotingKeys``

Reply:
`renewVotingKeys`?

Reply:
Options: 

1. updateVotingKeys
2. createVotingKeys
3. addVotingKeys
4. renewVotingKeys

maybe 2?

Reply:
This command, together with `link`, does two things, right? Creates new keys AND removes (or unregisters) the old ones. I think `add` and `create` both fail to convey the second action. I can see @Wayonb's point that the keys are not actually "updated", so I suggested the `renew` option :)
But this command will be very seldom used (twice a year!) so let's go with whatever :D

Reply:
I've talked to @Wayonb , updateVotingKeys is the name.